pages/community/dei-mission-statement.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: DEI Mission Statement
+subtitle: Diversity, Equality, and Inclusion at US-RSE
+permalink: /dei-mission/
+---
+
+The US-RSE Association is committed to providing an [inclusive environment with
+equitable treatment]({{site.url}}/code-of-conduct/) for all and to promoting
+and encouraging diversity throughout the RSE community in the US. At US-RSE,
+integrating DEI practices into our education programs, governance structure,
+and culture is at the forefront of our mission to ensure a welcoming,
+nurturing, and robustly inclusive community. We believe that the amplification
+of diverse perspectives is essential for driving innovation, promoting
+creativity, and encouraging engagement for the success of RSEs. We welcome and
+respect individuals of all dimensions of diversity, including but not limited
+to race, color, caste, economic status, gender expression, gender identity,
+sexual orientation, disability, age, religion (or lack thereof), national
+origin, and ethnicity.
```

Comment:
I think that's a good addition of a general category that we missed. @exoticDFT could you make that change? 
```suggestion
to race, color, caste, economic status, gender expression, gender identity,
sexual orientation, disability, neurocognitive differences, age, 
religion (or lack thereof), national origin, and ethnicity.
```

Reply:
Sounds great! Will do.